src/_components/ExploreManual.jsx

```diff
@@ -0,0 +1,109 @@
+export default function ExploreSection() {
+  return (
+    <section className="bg-background-grey py-12 md:py-24">
+      <div className="px-6 max-w-7xl mx-auto">
+        <h2 className="text-4xl font-bold text-green mb-12">
+          EXPLORE OUR SITE
+        </h2>
+
+        <div className="grid grid-cols-1 lg:grid-cols-5 gap-12 items-start">
+          <div className="lg:col-span-2 flex flex-col items-center lg:items-start">
+            <img
+              src="/assets/book.png"
+              alt="Manual Books"
+              className="w-full max-w-md h-auto mb-8"
+            />
+
+            <div className="flex items-center gap-6">
+              <h3 className="text-5xl font-bold text-primary">
+                READ THE
+                <br />
+                MANUAL
+              </h3>
+
+              <svg className="w-20 h-20" viewBox="0 0 64 64" fill="none">
+                <path
+                  d="M20 32L44 32M44 32L36 24M44 32L36 40"
+                  stroke="black"
+                  strokeWidth="3"
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                />
+              </svg>
+            </div>
+          </div>
+
+          <div className="lg:col-span-3 grid grid-cols-1 md:grid-cols-2 gap-6">
+            <a
+              href="/faqs"
+              className="bg-white rounded-lg p-6 flex flex-col items-center shadow-sm hover:shadow-lg transition-shadow"
+            >
+              <img
```

Comment:
You can prolly put these buttons into a component since all 4 look identical except for the actual text content and svg. Can you also increase the shadow effect on the component? maybe `shadow-xl` or `shadow-2xl`